docker-base-services.yml

```diff
@@ -38,6 +38,8 @@ services:
       - FLYWAY_BASELINE_ON_MIGRATE=true
       - FLYWAY_PLACEHOLDERS_api_db_username=fam_proxy_api
       - FLYWAY_PLACEHOLDERS_api_db_password=test
+      - FLYWAY_PLACEHOLDERS_admin_api_db_user=fam_proxy_admin_api
+      - FLYWAY_PLACEHOLDERS_admin_api_db_password=ojkr97y663kvqcwlm8hyvf8dalxd
```

Comment:
Both values are current intended for **temporary** only. I assume without giving a value the build will fail, so just gave them some value. The subsequent ticket when the user/password are setup on secret manager than these can be adjusted.

Reply:
This doesn't sound right. The creation of the secret in S3 I'd expect to be part of this effort? Although you are right that the api_db_password value is just a placeholder, but I suggest using the value 'test' rather than something that looks like a real password.

Reply:
The password is replaced with clear dummy value.